AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/CredentialStorage/CredentialStoreOperationClient.swift

```diff
@@ -11,9 +11,9 @@ import Security
 
 protocol CredentialStoreStateBehaviour {
```

Comment:
nit: Did we thought of renaming this?

Reply:
Yes.. Will do that as a separate PR of moving away from the word credential and use store..